tests/azure/nsg_test.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func genNsgName() string {
-	return fmt.Sprintf("klb-nsg-tests-%d", rand.Intn(1000))
+	return fmt.Sprintf("klb-nsg-tests-%d", rand.Intn(999999999))
```

Comment:
auheauheaheuaheuaheauheahuea

I want a uuid actually, but did not wanted to add a dependency and had a lot of bigger fish to fry. Lets add that cool uuid routine :-)